src/app/home/home.component.ts

```diff
@@ -69,28 +69,26 @@ export class HomeComponent {
        })
    }
   dbpEndpointsConsumtion() {
-    let apsURL="http://<ApsURL>/activiti-app/api/enterprise/profile";
-    let acsURL="http://<AcsURL>/alfresco/api/-default-/public/alfresco/versions/1/people/-me-";
     let accessToken = this.oauthService.getAccessToken();
     console.debug('access-token', accessToken);
     let myHeaders = new Headers();
     myHeaders.set("Authorization", "Bearer " + accessToken);
     let options = new RequestOptions({ headers: myHeaders });
-    this.http.get(apsURL,options).
+    this.http.get(this.aps.url,options).
```

Comment:
I like that you've started separating things into their own classes.

You could consider renaming these `ACS` and `APS` classes to something like `ACSPersonDetailsService` and `APSPersonDetailsService`, then moving `get` calls and data mapping into some sort of fetch method there, just to consolidate the logic into a single class.

At the moment those classes are a bit of a mix of config and result store, with the behavior mixed into this method.

I certainly won't hold up the PR for it, just mentioning as another possible approach.